tmp/build/patch-charts.sh

```diff
@@ -248,9 +261,7 @@ function patchTemplates() {
 \1\2/' ${HELM_DIR}/istio/charts/sidecarInjectorWebhook/templates/deployment.yaml
 
   # change the location of the healthCheckFile from /health to /tmp/health
-  if [[ "${COMMUNITY,,}" != "true" ]]; then
-    sed -i -e 's/\/health/\/tmp\/health/' ${HELM_DIR}/istio/charts/sidecarInjectorWebhook/templates/deployment.yaml
-  fi
+  sed -i -e 's/\/health/\/tmp\/health/' ${HELM_DIR}/istio/charts/sidecarInjectorWebhook/templates/deployment.yaml
```

Comment:
is this change supposed to go in this PR?

Reply:
Yes.  Community images don't work without it for some reason.  Health checks were failing on sidecar injector pod.

Reply:
yep:

```
Readiness probe failed: Error: fail on inspecting path /health: stat /health: no such file or directory 
```

Reply:
I was not asking whether this is valid or not. I was asking if this should belong to this PR or another. I'm happy with it, but it seems it's a different problem than this PR fixes.

Reply:
I don't know how you want to look at it, but I couldn't test the changes without this.  Picnic area/campground rule: leave it cleaner than you found it.